lib/metasploit/framework/login_scanner/redis.rb

```diff
@@ -100,6 +133,13 @@ def set_sane_defaults
           self.max_send_size      ||= 0
           self.send_delay         ||= 0
         end
+
+        # Creates anonymous credentials to be used for authentication against a 'Redis' database.
+        #
+        # @return [Metasploit::Famework::Credential]
+        def anonymous_credential
```

Comment:
Maybe either change this name to reflect it is a requirement for this call to work for any other username/password combination to work or throw some comments above?